provider/core/src/data_provider.rs

```diff
@@ -160,7 +160,7 @@ where
     for<'a> &'a <M::Yokeable as Yokeable<'a>>::Output: PartialEq,
 {
     fn eq(&self, _other: &Self) -> bool {
-        todo!()
+        todo!("#753")
```

Comment:
suggestion (nb): Also perhaps add a full issue link in a comment and a short explanation?

Reply:
This code is gone now.